frontend/src/components/Forms/reports/ReportSubmissions.js

```diff
@@ -17,6 +17,22 @@ const defaultProps = {
   mineReportSubmissions: [],
 };
 
+const updateSubmissionHandler = (mine_document_guid, props) => {
+  const fileToRemove = props.mineReportSubmissions[
+    props.mineReportSubmissions.length - 1
+  ].documents.filter((doc) => doc.mine_document_guid === mine_document_guid)[0];
+  let updatedSubmissions = props.mineReportSubmissions;
```

Comment:
Should this be a find() as there should only be on item in the list that this returns right? filter implies many.

Reply:
Changed and yeah this is a way better option.